tests/unit/templating.spec.ts

```diff
@@ -185,7 +185,7 @@ describe('Pipeline interpolator', () => {
       {
         name: 'fromdate',
         column: '<%= foo %>',
-        format: '<%= bar %>',
+        format: '<%= foo %>',
```

Comment:
Is that it important ? 🤔 Otherwise no worries !

Reply:
no it's not :) whatever we test that it does not try to innterpolate